teams.md

```diff
@@ -10,4 +10,13 @@
 
 ## Data Science
 
+The Data Science team works on the Intelligence Suite, which aims to use machine intelligence to unlock business value and drive engagement across channels by
+
+- Elevating the level of their regular tasks, freeing them from manually crafting marketing tactics as they begin to trust the intelligent automation more
```

Comment:
Who is them/their in this sentence? I think it's referring to marketers / our customers, but we haven't mentioned them yet so there is no clear reference.

Maybe edit the preceding section to replace "across channels" with "for our customers" or something like that. Then I think it all ties together.

Reply:
Good catch, I think I made some changes to the top without looking at the impact below, fixing..